### DISCUSSION TITLE
章节1.2.1 中的数据使用疑惑

### DISCUSSION THREAD

**Discussion by huisnotacouncillor** [isno/theByteBook#112] [General]
“十年前，当我在创办的 Netscape 公司时，大概 5000 千人使用了宽带互联网，而现在有超过 20 亿人使用宽带互联网。” 请问这段中的 5000 千人指的是 5000 人还是 5,000,000 人？如果是后者，翻译 500 万人更符合阅读习惯一些

**Comment by isno** [comment]
thanks， 是 500 万人，我修改一下。
